src/components/forms/news-form/news-form.js

```diff
@@ -147,65 +135,71 @@ const NewsForm = ({ id, newsArticle, editMode }) => {
     inputs
   };
 
+  const navButtons = (
+    <>
+      <BackButton />
+      <SaveButton
+        className={styles.saveButton}
+        data-cy='save'
+        type='submit'
+        title={SAVE_TITLE}
+      />
+    </>
+  );
+
   return (
     <div>
       <form onSubmit={handleSubmit}>
-        <div className={styles.controlsBlock}>
-          <div>{languageCheckboxes}</div>
-          <SaveButton
-            className={styles.saveButton}
-            data-cy='save'
-            type='submit'
-            title={SAVE_TITLE}
-          />
-        </div>
-        <Box my={3}>
-          <div className={styles.imageUploadAvatar}>
-            <ImageUploadContainer
-              handler={handleImageLoad}
-              buttonLabel={AUTHOR_PHOTO}
-            />
+        {navButtons}
```

Comment:
pls remove one this component

Reply:
Improved UI, there are a lot of inputs and you need to scroll a lot to go backwards or save the news.
As an option, I can make the position fixed as we have save/delete while editing a product.